src/julia.h

```diff
@@ -694,14 +708,57 @@ STATIC_INLINE jl_value_t *jl_cellset(void *a, size_t i, void *x)
 #define jl_data_ptr(v)  (((jl_value_t*)v)->fieldptr)
 
 // struct type info
-#define jl_field_offset(st,i)  (((jl_datatype_t*)st)->fields[i].offset)
-#define jl_field_size(st,i)    (((jl_datatype_t*)st)->fields[i].size)
-#define jl_field_isptr(st,i)   (((jl_datatype_t*)st)->fields[i].isptr)
 #define jl_field_name(st,i)    (jl_sym_t*)jl_svecref(((jl_datatype_t*)st)->name->names, (i))
 #define jl_field_type(st,i)    jl_svecref(((jl_datatype_t*)st)->types, (i))
 #define jl_datatype_size(t)    (((jl_datatype_t*)t)->size)
 #define jl_datatype_nfields(t) (((jl_datatype_t*)(t))->nfields)
 
+#define _jl_field_accessors(f)                                          \
```

Comment:
Julia.h is public so it should still be prefixed even if later undefed to reduce the chance of colliding with defines from embedding projects